authenticatorbridge/src/test/java/com/bitwarden/authenticatorbridge/util/FakeSymmetricKeyStorageProvider.kt

```diff
@@ -0,0 +1,13 @@
+package com.bitwarden.authenticatorbridge.util
+
+import com.bitwarden.authenticatorbridge.model.SymmetricEncryptionKeyData
+import com.bitwarden.authenticatorbridge.provider.SymmetricKeyStorageProvider
+
+/**
+ * A fake implementation of [SymmetricKeyStorageProvider] for testing purposes.
+ */
+class FakeSymmetricKeyStorageProvider : SymmetricKeyStorageProvider {
+
+    override var symmetricKey: SymmetricEncryptionKeyData? = null
+
```

Comment:
Can we remove the extra newlines on this to match `FakeLifecycleOwner`?

Reply:
955a1eb1b43d4cb1f6feb5c187845b70fad9e997